src/test/ui/type/type-annotation-needed.stderr

```diff
@@ -1,11 +1,13 @@
-error[E0283]: type annotations needed: cannot resolve `_: std::convert::Into<std::string::String>`
+error[E0283]: type annotations needed
   --> $DIR/type-annotation-needed.rs:6:5
    |
 LL | fn foo<T: Into<String>>(x: i32) {}
    |    ---    ------------ required by this bound in `foo`
 ...
 LL |     foo(42);
-   |     ^^^
+   |     ^^^ cannot infer type for `T`
+   |
+   = note: cannot resolve `_: std::convert::Into<std::string::String>`
```

Comment:
This should suggest turbofish:

    help: set the type `T` for this function call: `foo::<T>(42)`